spec.html

```diff
@@ -39345,6 +39139,14 @@ <h1>Properties of Generator Instances</h1>
               The execution context that is used when executing the code of this generator.
             </td>
           </tr>
+          <tr>
+            <td>
+              [[GeneratorBrand]]
+            </td>
+            <td>
+              The brand of the generator used to distinguish different type of built-in generators. [[GeneratorBrand]] of generators declared by the ECMAScript SourceText must be ~empty~.
```

Comment:
```suggestion
              The brand of the generator used to distinguish different type of built-in generators. [[GeneratorBrand]] of generators declared by ECMAScript source text must be ~empty~.
```
altho i'm not sure if this is the best way to phrase this.

also i'm not sure "brand" is the proper term to use here, but i don't have a better suggestion off the top of my head